sql/moz-fx-data-shared-prod/accounts_backend_external/dataset_metadata.yaml

```diff
@@ -2,10 +2,13 @@ friendly_name: Firefox Accounts Backend External
 description: |-
   Data extracted from the FxA backend services databases.
   See https://mozilla.github.io/ecosystem-platform/reference/database-structure#database-fxa for more information.
+
+  Access to this dataset is restricted to accounts-confidential workgroup because some tables here contain sensitive data.
+  `mozilla-confidential` user-facing views are available in the `accounts_backend` dataset.
 dataset_base_acl: restricted
 user_facing: false
 labels: {}
 workgroup_access:
-- role: roles/bigquery.dataEditor
+- role: roles/bigquery.dataViewer
   members:
-  - workgroup:platform-infra/internal
+  - workgroup:accounts-confidential
```

Comment:
This restricts access to the whole `accounts_backend_external` dataset to a group created in https://mozilla-hub.atlassian.net/browse/DSRE-1408.
non-confidential user-facing views are going to be deployed in `accounts_backend` dataset.

Reply:
IMO it'd be good for DEs who are working on these ETLs to still have Data Editor access here (e.g. we could have a custom subgroup like `accounts-confidential/writers`).

@whd thoughts?

Reply:
I mentioned this to akkomar as an option but IIRC we were in agreement that no direct write access would be preferable if possible and that we'd add it via a writers subgroup if it wound up being necessary. A general solution or pattern is the subject of https://mozilla-hub.atlassian.net/browse/DSRE-1023.

Reply:
I was expecting there will be simple 1-1 copy-data queries here, without any complex logic so write access didn't seem to be required to me.